pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -217,4 +215,41 @@ type AlertRuleGroup struct {
 
 // AlertingFileExport is the full provisioned file export.
 // swagger:model
-type AlertingFileExport = file.AlertingFileExport
+type AlertingFileExport struct {
+	APIVersion int64                  `json:"apiVersion" yaml:"apiVersion"`
+	Groups     []AlertRuleGroupExport `json:"groups" yaml:"groups"`
+}
+
+// AlertRuleGroupExport is the provisioned file export of AlertRuleGroupV1.
+type AlertRuleGroupExport struct {
+	OrgID    int64             `json:"orgId" yaml:"orgId"`
+	Name     string            `json:"name" yaml:"name"`
+	Folder   string            `json:"folder" yaml:"folder"`
+	Interval Duration          `json:"interval" yaml:"interval"`
+	Rules    []AlertRuleExport `json:"rules" yaml:"rules"`
+}
+
+// AlertRuleExport is the provisioned file export of models.AlertRule.
+type AlertRuleExport struct {
+	UID          string              `json:"uid" yaml:"uid"`
+	Title        string              `json:"title" yaml:"title"`
+	Condition    string              `json:"condition" yaml:"condition"`
+	Data         []AlertQueryExport  `json:"data" yaml:"data"`
+	DashboardUID string              `json:"dasboardUid,omitempty" yaml:"dashboardUid,omitempty"`
+	PanelID      int64               `json:"panelId,omitempty" yaml:"panelId,omitempty"`
+	NoDataState  NoDataState         `json:"noDataState" yaml:"noDataState"`
+	ExecErrState ExecutionErrorState `json:"execErrState" yaml:"execErrState"`
+	For          Duration            `json:"for" yaml:"for"`
```

Comment:
This was changed from upstream `model.Duration` to our own version, the semantics are not the same. It's the reason some tests are failing.

Reply:
fixed!